src/resources/js/views/manager.js

```diff
@@ -291,6 +306,42 @@ tribe.events.views.manager = {};
 		return false;
 	};
 
+	/**
+	 * Catches the normal browser interactions for Next and Previous pages
+	 * so that we can use the manager to load the page requested instead
+	 * of just chaning the URL.
+	 *
+	 * @since  TBD
+	 *
+	 * @param  {Event} event DOM Event related to the window popstate
+	 *
+	 * @return {boolean}     Will always return false on this one.
+	 */
+	obj.popstate = function( event ) {
+		var $container = $( obj.selectors.container );
+		var containerData = obj.getContainerData( $container );
+
+		// Flag that we are doing popstate globally.
+		obj.doingPopstate = true;
+
+		$container.trigger( 'beforePopState.tribeEvents', event );
+
+		var url = event.originalEvent.originalTarget.location.href;
+		var nonce = $container.data( 'view-rest-nonce' );
+		var shouldManageUrl = obj.shouldManageUrl( $container );
+
+		var data = {
+			url: url,
+			_wpnonce: nonce
+		};
+
+		obj.request( data, $container );
+
+		$container.trigger( 'afterPopState.tribeEvents', event );
```

Comment:
All of this should be in an `each()` in the case we have multiple containers right?